### ISSUE TITLE
Training accuracy of TF official tutorial is very low with Ubuntu 18.04, CUDA 10.0 and cuDNN 7.4

### ISSUE THREAD

**Issue by ToughLife** [tensorflow/tensorflow#25270]
**System information**
- Ubuntu 18.04
- Python 3.7.1 (Anaconda 3)
- CUDA 10.0
- cuDNN 7.4.2
- GTX 1070 mobile
- Core i7 8950H, 6C12T @2.2G Hz

**Describe the current behavior**
    I run the code from the TensorFlow tutorial from https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/basic_classification.ipynb, but the training accuracy is very low (~0.1). 

No matter I installed the TensorFlow from 1) compiling source or from 2) `pip install tf-nightly-gpu`, the accuracies are both around 0.1. The stable release of tensorflow is not available for Python 3.7. Besides, I have tested running the code on `GPU` and `CPU`, the results are same.

**Describe the expected behavior**
In the tutorial, the expected accuracy is around 0.8.

**Code to reproduce the issue**

```python
import tensorflow as tf
from tensorflow import keras
import os

USE_CPU = 1
if USE_CPU == 1:
    os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"  
    os.environ["CUDA_VISIBLE_DEVICES"] = ""

fashion_mnist = keras.datasets.fashion_mnist
(train_images, train_labels), (test_images, test_labels) = fashion_mnist.load_data()

train_images = train_images / 255.0
test_images = test_images / 255.0

model = keras.Sequential([
    keras.layers.Flatten(input_shape=(28, 28)),
    keras.layers.Dense(128, activation=tf.nn.relu),
    keras.layers.Dense(10, activation=tf.nn.softmax)
])


model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy']
              )
model.fit(train_images, train_labels, epochs=5)
test_loss, test_acc = model.evaluate(test_images, test_labels)
print('Test accuracy:', test_acc)
```

The output of the above code:
```
WARNING: Logging before flag parsing goes to stderr.
W0128 17:35:44.705833 140133976348480 deprecation.py:506] From /opt/anaconda3/lib/python3.7/site-packages/tensorflow/python/ops/init_ops.py:1253: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
W0128 17:35:44.735007 140133976348480 deprecation.py:506] From /opt/anaconda3/lib/python3.7/site-packages/tensorflow/python/keras/engine/base_layer_utils.py:123: calling Zeros.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
2019-01-28 17:35:44.862411: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-01-28 17:35:44.866879: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2208000000 Hz
2019-01-28 17:35:44.867464: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x5574ea7aa690 executing computations on platform Host. Devices:
2019-01-28 17:35:44.867481: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>
2019-01-28 17:35:44.868907: I tensorflow/stream_executor/platform/default/dso_loader.cc:154] successfully opened CUDA library libcuda.so.1 locally
2019-01-28 17:35:44.871695: E tensorflow/stream_executor/cuda/cuda_driver.cc:303] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2019-01-28 17:35:44.871714: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:161] retrieving CUDA diagnostic information for host: toughlife-AW17R5
2019-01-28 17:35:44.871719: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:168] hostname: toughlife-AW17R5
2019-01-28 17:35:44.871794: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:192] libcuda reported version is: 410.78.0
2019-01-28 17:35:44.871831: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:196] kernel reported version is: 410.78.0
2019-01-28 17:35:44.871836: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:303] kernel version seems to match DSO: 410.78.0
Epoch 1/5
60000/60000==============================] - 1s 22us/sample - loss: 0.5021 - acc: 0.1024
Epoch 2/5
60000/60000==============================] - 1s 21us/sample - loss: 0.3748 - acc: 0.1025
Epoch 3/5
60000/60000==============================] - 1s 21us/sample - loss: 0.3365 - acc: 0.1025
Epoch 4/5
60000/60000==============================] - 1s 21us/sample - loss: 0.3132 - acc: 0.1031
Epoch 5/5
60000/60000==============================] - 1s 21us/sample - loss: 0.2931 - acc: 0.1023
60000/60000==============================] - 1s 9us/sample - loss: 0.2689 - acc: 0.1015
```

**Other info / logs**

**Comment by jvishnuvardhan** [comment]
@ToughLife Please check your system and size of files. There is something wrong with your system or installation. I ran it on my desktop and got the following results.

runfile('/usr/local/google/home/vishnuvardhanj/Desktop/Vishnu/TensorFlow/User Codes for Testing/untitled4.py', wdir='/usr/local/google/home/vishnuvardhanj/Desktop/Vishnu/TensorFlow/User Codes for Testing')
Downloading data from https://storage.googleapis.com/tensorflow/tf-keras-datasets/train-labels-idx1-ubyte.gz
32768/29515 [=================================] - 0s 6us/step
Downloading data from https://storage.googleapis.com/tensorflow/tf-keras-datasets/train-images-idx3-ubyte.gz
26427392/26421880 [==============================] - 2s 0us/step
Downloading data from https://storage.googleapis.com/tensorflow/tf-keras-datasets/t10k-labels-idx1-ubyte.gz
8192/5148 [===============================================] - 0s 2us/step
Downloading data from https://storage.googleapis.com/tensorflow/tf-keras-datasets/t10k-images-idx3-ubyte.gz
4423680/4422102 [==============================] - 1s 0us/step
Epoch 1/5
60000/60000 [==============================] - 7s 118us/step - loss: 0.4985 - acc: 0.8259
Epoch 2/5
60000/60000 [==============================] - 7s 115us/step - loss: 0.3767 - acc: 0.8636
Epoch 3/5
60000/60000 [==============================] - 7s 118us/step - loss: 0.3354 - acc: 0.8769
Epoch 4/5
60000/60000 [==============================] - 7s 117us/step - loss: 0.3134 - acc: 0.8847
Epoch 5/5
60000/60000 [==============================] - 7s 117us/step - loss: 0.2949 - acc: 0.8916
10000/10000 [==============================] - 1s 63us/step
Test accuracy: 0.8635
Thanks!

**Comment by jvishnuvardhan** [comment]
I am closing this issue as the issue was not due to TF. Thanks!
